include/boost/histogram/detail/probit.hpp

```diff
@@ -0,0 +1,41 @@
+// Copyright 2022 Jay Gohil, Hans Dembinski
+//
+// Distributed under the Boost Software License, Version 1.0.
+// (See accompanying file LICENSE_1_0.txt
+// or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_HISTOGRAM_DETAIL_PROBIT_HPP
+#define BOOST_HISTOGRAM_DETAIL_PROBIT_HPP
+
+#include <cmath>
+
+# define PI 3.14159265358979323846  /* pi */
```

Comment:
We only use the preprocessor as last resort, because it is evil. Best is to just replace PI in the code below with a literal, see comment there. Please remove line 12.